docs/pages/reactive.md

```diff
@@ -147,3 +147,4 @@ private async Task<bool> AttachZoomWatcher()
     return await _view!.AwaitReactiveSingleWatchUpdate<bool>("view?.zoom > 20");
```

Comment:
This is pretty cool! If you did this waiting a lot, would it cause thread exhaustion or something strange?

Reply:
Yes, I could see this causing exhaustion or even wasm single-thread deadlock if used a lot. But I would think the other methods that are reusable (don't fire just once) are more likely to be heavily used, and those all use a callback pattern which would avoid issues. I would add a warning here, but I'm not _entirely_ sure there is an issue, or how exactly it would manifest.

Reply:
Sounds good, let’s just see how it goes!